.env.example

```
@@ -29,13 +29,12 @@ LOG_STACK=single
-MONGO_DB_HOST=localhost
-MONGO_DB_PORT=27017
-MONGO_DB_USERNAME=
-MONGO_DB_PASSWORD=
-MONGO_DB_DATABASE=bioreport
-MONGO_DB_URI=
+DB_CONNECTION=pgsql
+DB_HOST=localhost
+DB_PORT=5432
+DB_DATABASE=bioreport
+DB_USERNAME=bioreport
+DB_PASSWORD=supersecretpassword
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DB_PASSWORD key should go before the DB_PORT key

(UnorderedKey)